### ISSUE TITLE
Model flux fail

### ISSUE THREAD

**Issue by gty111** [xdit-project/xDiT#282]
```
ValueError: Attention Processor class FluxSingleAttnProcessor2_0 is not supported by xFuser
```
It seems that xDit didn't implement FluxSingleAttnProcessor2_0

**Comment by feifeibear** [comment]
You need use diffusers==0.30.3.
Could you please check the version of your diffusers?

**Comment by gty111** [comment]
> You need use diffusers==0.30.3. Could you please check the version of your diffusers?

The version of diffusers is 0.30.3

**Comment by gty111** [comment]
Fixed by #292
